packages/discovery-provider/ddl/migrations/0055_collection_track_relations.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS collection_track_relations (
+  collection_id INTEGER NOT NULL,
+  track_id INTEGER NOT NULL,
+  is_removed BOOLEAN NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (collection_id, track_id)
+);
+
+CREATE INDEX IF NOT EXISTS idx_collection_track_relations_collection_id ON collection_track_relations USING btree (collection_id, created_at);
```

Comment:
Is this not redundant with the primary key?

Reply:
i'm guessing the intention is to separate the composite key and index on collection_id but not sure why created_at is necessary as well